src/Autofac.Integration.SignalR/Autofac.Integration.SignalR.csproj

```diff
@@ -1,87 +1,37 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{CCC12CB7-6753-4772-9B04-83A0B2554991}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Autofac.Integration.SignalR</RootNamespace>
-    <AssemblyName>Autofac.Integration.SignalR</AssemblyName>
-    <FileAlignment>512</FileAlignment>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\Autofac.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFramework>net461</TargetFramework>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\..\</SolutionDir>
-    <RestorePackages>true</RestorePackages>
-    <TargetFrameworkProfile />
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE;CODE_ANALYSIS</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <AssemblyTitle>Autofac.Integration.SignalR</AssemblyTitle>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <Description>ASP.NET SignalR 2 Integration for Autofac</Description>
+    <Copyright>Copyright © 2014 Autofac Contributors</Copyright>
+    <Version>6.0.0.0</Version>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+    <FileVersion>6.0.0.0</FileVersion>
+    <DocumentationFile>bin\$(Configuration)\Autofac.Integration.SignalR.xml</DocumentationFile>
     <CodeAnalysisRuleSet>..\..\Full.ruleset</CodeAnalysisRuleSet>
     <RunCodeAnalysis>true</RunCodeAnalysis>
-    <DocumentationFile>bin\Debug\Autofac.Integration.SignalR.xml</DocumentationFile>
-    <Prefer32Bit>false</Prefer32Bit>
+    <Title>Autofac ASP.NET SignalR 2 Integration</Title>
+    <PackageId>Autofac.SignalR2</PackageId>
+    <PackageProjectUrl>https://autofac.org</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/autofac/Autofac.SignalR</RepositoryUrl>
+    <RepositoryType>github</RepositoryType>
+    <PackageIconUrl>https://cloud.githubusercontent.com/assets/1156571/13684110/16b8f152-e6bf-11e5-84ae-22c66c6d351a.png</PackageIconUrl>
```

Comment:
Probably want to change to PackageIcon to replace PackageIconUrl; see the example in the MVC Integration.

Reply:
This one looks like it's still pending.

Reply:
Addressed with 6309f8009fd5645666b9dd65e1e200e41fbb851a